molgenis-security/src/test/java/org/molgenis/security/twofactor/OTPServiceImplTest.java

```diff
@@ -26,11 +27,19 @@ public OTPService otpService()
 	@Autowired
 	private OTPService otpService;
 
+	@Test(expectedExceptions = InvalidVerificationCodeException.class)
+	public void testTryVerificationKeyFailed()
+	{
+		boolean isValid = otpService.tryVerificationCode("", "secretKey");
+		assertEquals(false, isValid);
+	}
+
 	/**
 	 * FIXME(SH): how do we test this properly?
 	 */
```

Comment:
I propose to not test it because we would simply be testing the TOTP library here.